e2e/davinci-app/main.ts

```diff
@@ -184,6 +202,13 @@ const urlParams = new URLSearchParams(window.location.search);
           formEl, // You can ignore this; it's just for rendering
           collector, // This is the plain object of the collector
         );
+      } else if (collector.type === 'PhoneNumberCollector') {
```

Comment:
i know this doesn't have to be separated so we can remove this and add the if condition above but i just separated it for reading